seqr/views/apis/report_api_tests.py

```diff
@@ -1120,11 +1117,10 @@ def test_family_metadata(self):
         test_row = next(r for r in response_json['rows'] if r['familyGuid'] == 'F000012_12')
         self.assertDictEqual(test_row, {
             'projectGuid': 'R0003_test',
-            'internal_project_id': 'Test Reprocessed Project',
             'familyGuid': 'F000012_12',
             'family_id': '12',
             'displayName': '12',
-            'solve_status': 'Unsolved',
+            'solve_status': 'Partially solved',
```

Comment:
this is actually a more accurate way to show solve status, and `internal_project_id` was never used on the family metadata page